.github/workflows/ci-gh-cpu-build-and-test.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     with:
       build-target: cpu
       repos-name: ${{ github.event.repository.name }}
-      runs-on: ${{ contains(github.repository, 'nv-legate/legate.core') && 'linux-amd64-cpu4' || 'ubuntu-latest' }}
+      runs-on: ${{ contains(github.repository, 'nv-legate/legate.core') && 'linux-amd64-32cpu' || 'ubuntu-latest' }}
```

Comment:
Is the name of the runner misspelled? Should it be cpu32?

Reply:
No, the name is correct. CPU 4 is the 4th CPU while a 32 CPU machine has 32 CPUs. The linux-amd64-cpu4 label is incorrect.